clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClientSpec.groovy

```diff
@@ -41,9 +41,9 @@ class DockerRegistryClientSpec extends Specification {
 
   void "DockerRegistryClient should validate that it is pointing at a v2 endpoint."() {
     when:
-      Boolean result = client.isV2()
+      client.isV2()
 
     then:
-      result
+      true
```

Comment:
I can't really see what this test tests then. Seems like there should a version that passes, and a version that fails? Or is it demonstrating something else?